openmetadata-ui/src/main/resources/ui/src/styles/components/select.less

```diff
@@ -50,3 +50,10 @@
     }
   }
 }
+
+.ant-select.trim-select {
+  .ant-select-selection-overflow {
+    height: 200px;
```

Comment:
@chirag-madlani, this will render the select box with a fixed height of 200px, regardless of its content, right?

Reply:
Let me update it to max-height. thanks for pointing it out